deployment/roles/oracle/tasks/logging.yml

```diff
@@ -1,41 +1,9 @@
 ---
-- name: Slurp docker compose file
-  slurp:
-    src: "{{ bridge_path }}/oracle/docker-compose.yml"
-  register: docker_compose_slurp
-
-- name: Parse docker compose file
-  set_fact:
-    docker_compose_parsed: "{{ docker_compose_slurp['content'] | b64decode | from_yaml }}"
-
-- name: Set logger to remote server
-  set_fact:
-    docker_compose_parsed: "{{ docker_compose_parsed |combine({'services': {item: {'logging': {'driver': 'syslog','options': {'tag': '{{.Name}}/{{.ID}}'}}}}}, recursive=True) }}"
-  with_items: "{{ docker_compose_parsed.services }}"
-
-- name: Write new docker-compose file
-  copy:
-    content: "{{ docker_compose_parsed | to_yaml }}"
-    dest: "{{ bridge_path }}/oracle/docker-compose.yml"
-
-- name: Slurp docker compose extended file
-  slurp:
-    src: "{{ bridge_path }}/oracle/docker-compose-transfer.yml"
-  register: docker_compose_extended_slurp
-
-- name: Parse docker compose extended file
-  set_fact:
-    docker_compose_extended_parsed: "{{ docker_compose_extended_slurp['content'] | b64decode | from_yaml }}"
-
-- name: Set logger to remote server for extended file
-  set_fact:
-    docker_compose_extended_parsed: "{{ docker_compose_extended_parsed |combine({'services': {item: {'logging': {'driver': 'syslog','options': {'tag': '{{.Name}}/{{.ID}}'}}}}}, recursive=True) }}"
-  with_items: "{{ docker_compose_extended_parsed.services }}"
-
-- name: Write new docker-compose-extended file
-  copy:
-    content: "{{ docker_compose_extended_parsed | to_yaml }}"
-    dest: "{{ bridge_path }}/oracle/docker-compose-transfer.yml"
+- include_tasks: logging_remote.yml
```

Comment:
in fact, the actions performing by this task are configuring the docker-compose facility to forward logs to the syslog service on the host machine. Do you think it make sense to rename the file in `logging_by_syslog.yml`?

Reply:
Sure, renamed https://github.com/poanetwork/tokenbridge/pull/244/commits/a4eb446f7bf7571725261a277e6e3fdf6529a98e